cmd/nvidia-container-runtime-hook/main.go

```diff
@@ -114,9 +114,19 @@ func doPrestart() {
 	}
 	args = append(args, "configure")
 
-	if !hook.Features.AllowCUDACompatLibsFromContainer.IsEnabled() {
+	switch cli.CUDACompatMode {
+	case config.CUDACompatModeLdconfig:
+		args = append(args, "--cuda-compat-mode="+config.CUDACompatModeLdconfig)
+	case config.CUDACompatModeMount:
+		args = append(args, "--cuda-compat-mode="+config.CUDACompatModeMount)
+	case config.CUDACompatModeDisabled, config.CUDACompatModeHook:
 		args = append(args, "--no-cntlibs")
+	default:
+		if !hook.Features.AllowCUDACompatLibsFromContainer.IsEnabled() {
+			args = append(args, "--no-cntlibs")
+		}
```

Comment:
It feels cleaner to use the `--cuda-compat-mode=disabled` for the disabled case, than to use the "old" flag?

Reply:
That's true. I wanted to make it clearer that we are not changing behaviour, but I think I agree that this makes it cleaner.